doc/resources/jenkins/Jenkinsfile-acr

```diff
@@ -0,0 +1,32 @@
+node {
+  stage('init') {
+    // git clone
+    checkout scm
+  }
+
+  stage('build') {
+    sh 'mvn clean package'
+    acrQuickBuild azureCredentialsId: env.AZURE_CRED_ID,
+                  resourceGroupName: env.RES_GROUP,
+                  registryName: env.ACR_NAME,
+                  platform: "Linux",
+                  dockerfile: "Dockerfile",
+                  imageNames: [[image: "$env.ACR_REGISTRY/$env.IMAGE_NAME:$env.BUILD_NUMBER"]]
+  }
+
```

Comment:
One last thing - can  you add a stage ('test') to indicate testing should be adding... can be in comments